snapshots/artificial_intelligence/2024-06-03/epoch.csv.dvc

```diff
@@ -0,0 +1,36 @@
+# Learn more at:
+# http://docs.owid.io/projects/etl/architecture/metadata/reference/origin/
+meta:
+  origin:
+    # Data product / Snapshot
+    title: Parameter, Compute and Data Trends in Machine Learning
+    date_published: 2024-06-03
+    description_snapshot: |
+      We update this chart with the latest available data from our source every month.
+
+      The authors selected the AI systems for inclusion based on the following necessary criteria:
+      — Have an explicit learning component
+      — Showcase experimental results
+      — Advance the state of the art
+
+      In addition, the systems had to meet at least one of the following notability criteria:
+      — Paper has more than 1000 citations
+      — Historical importance
+      — Important state-of-the-art advance
+      — Deployed in a notable context
+
+      The authors note that: "For new models (from 2020 onward) it is harder to assess these criteria, so we fall back to a subjective selection. We refer to models meeting our selection criteria as 'milestone models."
+    # Citation
+    producer: Epoch
+    citation_full: 'Epoch, ‘Parameter, Compute and Data Trends in Machine Learning’. Published online at epochai.org. Retrieved from: ‘https://epochai.org/mlinputs/visualization’ [online resource]'
+    # Files
+    url_main: https://epochai.org/mlinputs/visualization
+    url_download: https://epochai.org/data/epochdb/all_systems.csv
+    date_accessed: 2024-06-03
+    # License
+    license:
+      name: Public domain
```

Comment:
please add a link to the license. If none available, please use a generic (https://creativecommons.org/public-domain/cc0/)

Reply:
just took a look and it is actually CC BY 4.0 license so I'll change it! thank you!